### PR TITLE
updating to latest mohawk interface for seen_nonce()

### PR DESCRIPTION
**Pull request by jcwilson**
The tests don't pass yet (unless one monkey-patches the mohawk library pulled down by tox), since the mohawk dependency version hasn't been updated yet.
Once the new version of mohawk is available on PyPi, i'll update this pull request.

### CODE REVIEW

File: hawkrest/__init__.py [hunk 1]
```diff
@@ -74,17 +74,17 @@ def lookup_credentials(cr_id):
...
     """
     Returns True if the Hawk nonce has been seen already.
     """
-    key = '{n}:{ts}'.format(n=nonce, ts=timestamp)
+    key = '{id}:{n}:{ts}'.format(id=id, n=nonce, ts=timestamp)
     if cache.get(key):
-        log.warning('replay attack? already processed nonce {k}'
-                    .format(k=key))
+        log.warning('replay attack? already processed nonce {k} for {id}'
+                    .format(k=key, id=id))
```
Comment (kumar303) [inline hawkrest/__init__.py]: because the ID is in the cache key, I don't think it's necessary to log it.

File: hawkrest/__init__.py [hunk 2]
```diff
@@ -74,17 +74,17 @@ def lookup_credentials(cr_id):
...
         return True
     else:
-        log.debug('caching nonce {k}'.format(k=key))
+        log.debug('caching nonce {k} for {id}'.format(k=key, id=id))
```
Comment (kumar303) [inline hawkrest/__init__.py]: same thing here

**Comment by kumar303** [comment]
Cool, the tests are working :) I'll get a release on PyPI soon.

**Comment by kumar303** [comment]
I just cut 0.3.0 and updated PyPI http://mohawk.readthedocs.org/en/latest/#changelog Thanks again for moving these changes through. You can bump the version requirement here: https://github.com/kumar303/hawkrest/blob/master/setup.py#L25

File: hawkrest/__init__.py (hunk 1 above)
Comment (jcwilson) [inline hawkrest/__init__.py]: good point, i'll fix it

**Comment by kumar303** [comment]
Looks great, thanks again!

**Comment by kumar303** [comment]
new version is up: https://pypi.python.org/pypi/hawkrest

**Comment by jcwilson** [comment]
Thanks for your help with this! Glad I could contribute back a bit. :)

**Pull request merged**